app/javascript/react/components/RegionShowContainer.js

```diff
@@ -20,13 +20,14 @@ const RegionShowContainer = (props) => {
       .then((response) => response.json())
       .then((body) => {
         let flavorList = body;
-        setFlavors(flavorList);
+        setFlavors(flavorList.region.flavors);
+        setRegionId(regionId);
       })
       .catch((error) => console.error(`Error in fetch: ${error.message}`));
   }, []);
 
   const listFlavors = getFlavors.map((flavor) => {
-    return <FlavorTile key={flavor.id} region={flavor.region} name={flavor.name} />;
+    return <FlavorTile key={flavor.id} name={flavor.name} flavorId={flavor.id} regionId={getRegionId}/>;
   });
 
   return <div>{listFlavors}</div>;
```

Comment:
shouldnt your region state also have the region's name?

Reply:
Made suggested changes